src/Microsoft.Diagnostics.EventFlow.Outputs.HttpOutput/Configuration/HttpOutputConfiguration.cs

```diff
@@ -0,0 +1,39 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.Diagnostics.EventFlow.Configuration
+{
+    // !!ACTION!!
+    // If you make any changes here, please update the README.md file to reflect the new configuration
+    public class HttpOutputConfiguration: ItemConfiguration
+    {
+        public static readonly string DefaultFormat = "json";
+
+        public string ServiceUri { get; set; }
+        public string Format { get; set; }
```

Comment:
Suggest defining an enumeration for this property and letting the configuration framework validate that a valid value is specified in the configuration file. E.g. see how TraceInput does this